src/dodal/devices/attenuator_base.py

```diff
@@ -0,0 +1,15 @@
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.core import epics_signal_r, epics_signal_rw, epics_signal_x
+
+
+class AttenuatorBase(StandardReadable):
+    """An attenuator base class with a minimum set of common PVs to phase1 beamlines."""
```

Comment:
Should: I think it's worth making clear here that the read transmission will be fractional i.e. 0-1 not a percentage